src/server/parser.ts

```diff
@@ -35,13 +35,13 @@ function startIndex() {
   )
 
   scanProcess.on('message', (mess) => {
-    const { data, log } = mess
+    const { data, msglog } = mess
```

Comment:
should also change the name from the scan emit

Reply:
Indeed, it works after rename.